spec/views/users/phone_confirmation/show.html.slim_spec.rb

```diff
@@ -47,4 +47,32 @@
       expect(rendered).to have_xpath("//input[@value='12777']")
     end
   end
+
+  context 'when choosing to receive OTP via SMS' do
+    before do
+      @sms_enabled = true
+      @fallback_confirmation_link = '/otp/new?delivery_method=voice'
```

Comment:
This seems strange.  Phone confirmation should be done via `users/phone_confirmation#send_code` no?   `otp/new` should be for logging in no?